scripts/popup.js

```diff
@@ -23,6 +24,23 @@ function view_all_function(){
 	document.location.href = open_url;
 }
 
+function search_tweet_function(){
+	var twitter_url = "https://twitter.com/search?q=";
+	var url_toSearch = document.getElementById('tweet_URL_input').value;
+	var from_date = document.getElementById('from-date').value;
+	var to_date = document.getElementById('to-date').value;
+
+	if(!from_date) {
+		from_date = "2006-03-21";	
+	}
+	
+	if (!to_date) {
+		var today = new Date();
+		to_date = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+today.getDate();
+	}
+
+	chrome.tabs.create({ url: twitter_url+"\""+url_toSearch+"\"%20since%3A"+from_date+"%20until%3A"+to_date });
+
```

Comment:
@gureks You have tried to do a great job but unfortunately your version of the extension is not working and giving an error to console. You have missed a closing `}` here because of which this function doesn't do anything on clicking the search button. I hope you will rectify this soon.